src/apis/nimbus.js

```diff
@@ -104,6 +120,93 @@ var nimbus = class extends ExtensionAPI {
               throw error;
             }
           },
+
+          async evaluateJEXL(expression, context = {}) {
+            try {
+              const targetingContext = await new TargetingContext(context, {});
+
+              const result = await targetingContext.evalWithDefault(expression);
+
+              return result;
+            } catch (error) {
+              console.error("Error evaluating expression:", error);
+              throw error;
+            }
+          },
+
+          async getClientContext() {
+            function isDate(object) {
```

Comment:
Can we add a comment explaining why we're doing this (because the Dates come from a different JS compartment and thus have a different Date constructor) and why this works? e.g., Date method calls need a `[[DateValue]]` internal slot and only Date objects will have that slot).